src/Sentry/Span.cs

```diff
@@ -88,6 +100,7 @@ public Span(ISpan tracer)
         Status = tracer.Status;
         IsSampled = tracer.IsSampled;
         _extra = tracer.Extra.ToDictionary();
+        _measurements = tracer.Measurements.ToDictionary();
```

Comment:
```suggestion
        InternalMeasurements = tracer.Measurements?.ToDictionary();
```
So then we can keep it `null` if there is nothing to convert over.